pkgs/by-name/ha/hamrs-pro/update.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash nix-update common-updater-scripts nix
+
+set -euo pipefail
+
+currentVersion=$(nix-instantiate --eval -E "with import ./. {}; hamrs-pro.version or (lib.getVersion hamrs-pro)" | tr -d '"')
+latestVersion=$(curl -sL https://hamrs.app | grep -E 'Download v[0-9]+\.[0-9]+\.[0-9]+' | grep -o -E '[0-9]+\.[0-9]+\.[0-9]+')
```

Comment:
Just grepping for the version anywhere within the HTML is not great. Please find a better source of the version string that is less brittle to any changes on the project's homepage.

Reply:
There's no open source release nor API to detect the latest version. I would suggest that we not include an update.sh.

Reply:
@ethancedwards8 and @jhollowe, what do you think? Ultimately, you're the ones who get to maintain this.

Reply:
Suggestion: grep on `https://hamrs-dist.s3.amazonaws.com/hamrs-pro-([0-9.]+)` and extract the version number in parentheses. That's a bit safer than expecting "download v xx.xx.x` text on the page.

@jhollowe is this an acceptable compromise?

Reply:
yeah, extracting the version from the download link should be good enough.